### DISCUSSION TITLE
Clean cache

### DISCUSSION THREAD

**Discussion by mechrekt** [immich-app/immich#7434] [Q&A]
Hi, I removed several photos and videos from my collection and removed offline files, but the thumbnail cache and encoded-video folders still contains the old files.
Is there a way to cleanup those folders?

Thank you.

**Comment by pedrojreis** [comment]
might be related to #4272 or #2748 .

What I mean is that when deleting an image it should also cleanup/invalidate cache, but it's currently bugged and a known bug :)

**Reply by AlmightyFrog** [reply]
Issue seems still not being fixed.
I restructured part of my library and now not only all thumbs and transcode (ffmpeg seems to not use hw transcode on my igpu) runs again, but also my disk gets dangerously full cause the "old" thumbs and transcdoe seem not to be deleted.

Is there a timelime for merging the proposed fix in linked tickets?
